### PR TITLE
main/truecrypt: remove

### PR DESCRIPTION
**Pull request by maxice8**
- Unmaintained since May 2014
- VeraCrypt is recommended alternative

See: http://truecrypt.sourceforge.net/

### CODE REVIEW

File: community/faenza-icon-theme/APKBUILD [hunk 1]
```diff
@@ -63,11 +59,9 @@ subpackages="$pkgname-extras
 
 source="https://launchpad.net/~tiheum/+archive/equinox/+files/faenza-icon-theme_$pkgver.tar.gz"
 
-_builddir="$srcdir"/$pkgname-1.3
+builddir="$srcdir"/$pkgname-1.3
```
Comment (tcely) [inline community/faenza-icon-theme/APKBUILD]:
```suggestion
builddir="$srcdir"/$pkgname-${pkgver%.[0-9]}
```

Comment (maxice8) [reply community/faenza-icon-theme/APKBUILD]: done

**Comment by ncopa** [comment]
please rebase

**Comment by maxice8** [comment]
On Mon, May 27, 2019 at 11:26:45AM -0700, Natanael Copa wrote:
> please rebase
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8167#issuecomment-496281021

Done

**Pull request closed without merging**